engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/rollingformulamulticolumn/RollingFormulaMultiColumnOperator.java

```diff
@@ -195,6 +221,7 @@ public void reset() {
             for (RingBufferWindowConsumer consumer : inputConsumers) {
                 consumer.reset();
             }
+            keyValuesSet = false;
```

Comment:
Remind me, when do we reset? Does it happen between buckets?

Reply:
Yes, the contexts are one-per-thread and will be reset and re-used each bucket.